crates/oxc_sourcemap/Cargo.toml

```
@@ -19,11 +19,11 @@ workspace = true
 doctest = false
 
 [dependencies]
-rustc-hash  = { workspace = true }
-serde       = { workspace = true, features = ["derive"] }
-serde_json  = { workspace = true }
-base64-simd = { workspace = true }
-cfg-if      = { workspace = true }
+rustc-hash   = { workspace = true }
+serde        = { workspace = true, features = ["derive"] }
+serde_json   = { workspace = true }
+base64-simd  = { workspace = true }
+cfg-if       = { workspace = true }
```

Remove from diff (if you can be bothered).

```suggestion
rustc-hash  = { workspace = true }
serde       = { workspace = true, features = ["derive"] }
serde_json  = { workspace = true }
base64-simd = { workspace = true }
cfg-if      = { workspace = true }
```